inc/events-calendar.php

```diff
@@ -39,7 +38,7 @@ function nightingale_widgets_events() {
  * @return array
  */
 function nightingale_remove_end_time_single( $formatting_details ) {
-	$formatting_details['show_end_time'] = 0;
+	$formatting_details[ show_end_time ] = 0;
```

Comment:
That is invalid PHP Array structure? how does that `formatting_details[ show_end_time ]` work?

Reply:
urk. This needs fixing. Thanks.